homeassistant/components/knx.py

```diff
@@ -198,7 +206,7 @@ def update(self):
             return False
 
 
-class KNXMultiAddressDevice(KNXGroupAddress):
+class KNXMultiAddressDevice(Entity):
```

Comment:
A KNXGroupAddress has one main address, but a MultiAddressDevice hasn't. If you look at the code, the logic isn't duplicated as the MultiAddress device does not have any internal state. It only provides the basic methods for inherited classes.
An example if a thermostat: It has 2 addresses (target and current temperature), but none of them would be the main address of the device.